app/mailers/notification_mailer.rb

```diff
@@ -1,4 +1,7 @@
 class NotificationMailer < ActionMailer::Base
+  include CMS::EmailTemplate::MailerExtension
```

Comment:
So this class is only for Provider

Provider users can choose which notification they get in their notification preferences
Developers do not really have this option.

@macejmic can we have a way to use `subscribe_for_notification` in another class?

Reply:
not really, we would need to create the same logic for developers.